### ISSUE TITLE
PyPI release 0.5 is unusable

### ISSUE THREAD

**Issue by scottbelden** [3601314/hbase-python#5]
The release doesn't include the changes to the setup.py requirements that were added [here](https://github.com/3601314/hbase-python/commit/f2b6a2b3191cd12ca2d323a51f6391399591c345). This means that after installing, if you try to `import hbase` you get an error about not having `protobuf` installed.

**Comment by senegal8green** [comment]
@scottbelden 'pip install protobuf'  did a trick in my case

**Comment by scottbelden** [comment]
Yes, that will work, but it might not be obvious to someone that they need to do that.
